openapi.yaml

```diff
@@ -0,0 +1,828 @@
+openapi: 3.0.0
+info:
+  version: 0.0.0
+  title: Conquery API
+  description: Conquery API
+  license:
+    #TODO find a fitting license
+    name: TODO
+    url: http://TODO/
+tags:
+  - name: dataset
+  - name: config
+  - name: query
+  - name: entity-preview
+  - name: concept
+  - name: frontend
+  - name: user
```

Comment:
ah gut zu wissen, ich wusste nicht was das macht, aber es legt die Reihenfolge in der UI fest.

Reply:
Ha, danke dann sortier ich das gleich mal ein wenig :D